### PR TITLE
upgrade citizen chart

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

upgrade to use latest citizen chart

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ x] No

### CODE REVIEW

File: charts/cmc-claim-store/values.preview.template.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ postgresql:
   initdbScripts:
     init.sql: |-
       CREATE DATABASE cmc WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "claimstore" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
```
Comment (sabahirfan) [inline charts/cmc-claim-store/values.preview.template.yaml]: we don't need `claimstore` database any more to run claim-store-api.

File: charts/cmc-claim-store/values.preview.template.yaml [hunk 2]
```diff
@@ -37,12 +38,15 @@ java:
...
-    DOC_ASSEMBLY_URL: http://dg-docassembly-aat.service.core-compute-aat.internal
+    DOC_ASSEMBLY_URL: false
     FRONTEND_BASE_URL: https://www-${SERVICE_FQDN}
     RESPOND_TO_CLAIM_URL: https://www-${SERVICE_FQDN}/first-contact/start
     CMC_DB_HOST: ${SERVICE_NAME}-postgres
     CMC_DB_USERNAME: hmcts
     CMC_DB_PASSWORD: hmcts
+    CLAIM_STORE_DB_HOST: ${SERVICE_NAME}-postgres
+    CLAIM_STORE_DB_USERNAME: hmcts
+    CLAIM_STORE_DB_PASSWORD: hmcts
```
Comment (sabahirfan) [inline charts/cmc-claim-store/values.preview.template.yaml]: we don't need `claimstore` database any more to run claim-store-api. So these are also not required anymore.

File: charts/cmc-claim-store/values.yaml [hunk 3]
```diff
@@ -33,6 +33,8 @@ java:
     CLAIM_STORE_TEST_SUPPORT_ENABLED: true
     CMC_DB_NAME: cmc
     CMC_DB_PORT: 5432
+    CLAIM_STORE_DB_NAME: claimstore
+    CLAIM_STORE_DB_PORT: 5432
```
Comment (sabahirfan) [inline charts/cmc-claim-store/values.yaml]: same here

File: charts/cmc-claim-store/values.preview.template.yaml (hunk 1 above)
Comment (klawani) [reply charts/cmc-claim-store/values.preview.template.yaml]: Removing it has caused lot of disruptions on product chart level especially for claim-store pod. I prefer to reinstate it there for the time being.

File: charts/cmc-claim-store/values.preview.template.yaml (hunk 2 above)
Comment (klawani) [reply charts/cmc-claim-store/values.preview.template.yaml]: Same here ... reinstating to avoid disruptions observed on product chart level.

**Pull request closed without merging**